kratos/sources/variables.cpp

```diff
@@ -160,6 +160,10 @@ KRATOS_CREATE_VARIABLE(double, LAGRANGE_AIR_PRESSURE)
 KRATOS_CREATE_VARIABLE(double, LAGRANGE_WATER_PRESSURE)
 KRATOS_CREATE_VARIABLE(double, LAGRANGE_TEMPERATURE)
 
+KRATOS_CREATE_VARIABLE(double, ARC_LENGTH_LAMBDA)
+KRATOS_CREATE_VARIABLE(double, ARC_LENGTH_RADIUS_FACTOR)
+KRATOS_CREATE_VARIABLE(bool, IS_CONVERGED)
```

Comment:
please don't add variables to the core that are not used there
unless they are very general (like e.g. `DISPLACEMENT`)

Reply:
Where is the arc length BTW?